nucypher/characters/chaotic.py

```diff
@@ -108,7 +108,9 @@ def status():
         # Server
         #
 
-        deployer = HendrixDeploy(action="start", options={"wsgi": rest_app, "http_port": http_port})
+        deployer = HendrixDeployWithStatics(action="start", options={"wsgi": rest_app, "http_port": http_port})
+        deployer.resources = get_statics()
```

Comment:
Can this call be embedded inside of the above `HendrixDeployWithStatics` init?

Reply:
I left it there because if we merge those changes in hendrix, then statics will be added like this
```
HendrixDeploy(action="start", options={"wsgi": rest_app, "http_port": http_port, "resources": get_statics()})
```